tools/profiles/mprintf.json

```diff
@@ -0,0 +1,8 @@
+{
+    "GCC_ARM": {
```

Comment:
I was not aware you could specify multiple `--profile` on the CLI. That's cool.

Why no ARMC6/IAR here? You seem to have the intercepts.

Reply:
No addition is required in the profile for ARMC6/IAR.
All that is required is implemented by mbed_printf_wrapper.c